napari/utils/perf/_stat.py

```diff
@@ -57,5 +57,5 @@ def average(self) -> int:
         average value : int.
         """
         if self.count > 0:
-            return self.sum / self.count
+            return self.sum // self.count
```

Comment:
It super weird to have a in average... but it does seem to be what the class is supposed to do...